### PR TITLE
Add xenial build

### PR DESCRIPTION
**Pull request by kvlou**
Use docker to build the packages, so we can build xenial and trusty packages.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,51 +1,74 @@
...
       sudo /etc/init.d/postgresql stop # stop postgresql again before installing the server
-      sudo apt-get install postgresql-$PG_SUPPORTED_VERSIONS
+      apt-get install -y postgresql-$PG_SUPPORTED_VERSIONS
     fi
-  # stop the travis-provided cluster
-  - sudo /etc/init.d/postgresql stop
-  - pg_lsclusters
-  - dpkg -l postgresql\* | cat
+
+    /etc/init.d/postgresql stop
```
Comment (jdowning) [inline .travis.yml:45]: does this work on xenial?

File: .travis.yml [hunk 2]
```diff
@@ -1,51 +1,74 @@
...
+    pg_lsclusters
+    dpkg -l postgresql\* | cat
```
Comment (jdowning) [inline .travis.yml:47]: i don't think you need to pipe to `cat` here if you're just displaying the output of the command

File: .travis.yml [hunk 3]
```diff
@@ -1,51 +1,74 @@
...
+
+    cd /root/build/pgextwlist
+
+    pg_buildext updatecontrol
+    dpkg-buildpackage -us -uc -rfakeroot
+    for deb in ../*.deb; do echo "$deb:"; dpkg-deb --info $deb; dpkg-deb --contents $deb; done
+    sudo dpkg -i ../*.deb
+    pg_buildext -i '--locale=C.UTF-8' -o local_preload_libraries=pgextwlist -o extwlist.extensions=citext,earthdistance,pg_trgm installcheck
+
+    exit 0
```
Comment (jdowning) [inline .travis.yml]: should we force `exit 0` here? if `pg_buildext` exits non-zero, we probably don't want it to package things up.

File: .travis.yml (hunk 1 above)
Comment (kvlou) [reply .travis.yml:45]: Seems to, I ran this in the xenial docker container:

```
root@3fa8a157b201:/# cat /etc/lsb-release ; ls /etc/init.d/postgresql ; /etc/init.d/postgresql stop
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.4 LTS"
/etc/init.d/postgresql
 * Stopping PostgreSQL 10 database server                                                                                                                                                                                               [ OK ]
root@3fa8a157b201:/#
```

File: .travis.yml (hunk 2 above)
Comment (kvlou) [reply .travis.yml:47]: This came form the original .travis.yml, i'm keeping it as is.

File: .travis.yml (hunk 3 above)
Comment (kvlou) [reply .travis.yml]: Probably not I will remove it.

**Review by genslein — APPROVED** [review]
to less crappy builds!

**Review by jdowning — APPROVED** [review]
:shipit:

**Comment by kvlou** [comment]
[pgextwlist + credentials vuln remediation](https://trello.com/c/VoMqgNHJ/2248-pgextwlist-credentials-vuln-remediation)

**Pull request merged**